po/de.po

```diff
@@ -1147,12 +1178,20 @@ msgid ""
 "Your help and involvement is what makes this software better. Feedback, bug "
 "reports and ideas are welcome... and patches even more so!"
 msgstr ""
+"Es ist bekannt, dass GTG k̶ä̶f̶e̶r̶f̶r̶e̶i̶ fehlerfrei ist! 🐛\n"
+"Jedoch, im Dunkeln können unerwartete Dinge geschehen...\n"
+"\n"
+"Wenn Sie jedoch Abstürze oder Fehlverhalten feststellen, dann melden Sie es "
```

Comment:
"jedoch" is repeated, use another word or remove it.

Reply:
Changed it to "aber"